### ISSUE TITLE
Feature Request: Resizable Image in editor

### ISSUE THREAD

**Issue by MegazeroMZ** [psergus/ngWYSIWYG#47]
Right now the only option we have for resize image is by taking over the insertImage function and add style="width:xx%" into the html we are going to insert into the editor. But is there a way to do it more dynamically like with textAngular (http://textangular.com/) where they have some kind of directive or something that allow click on Image and then a small box show up contain the image and can be drag around to resize the image? An alternative I'm using right now is instead of using prompt I create a modal and let the user input the size they want in text, and it just isnt intuitive enough.

**Comment by brunokrebs** [comment]
I have just released this feature under the [tag v0.6.0](https://github.com/psergus/ngWYSIWYG/releases/tag/v0.6.0). Please, take a look at it and, if possible, provide some feedback about usability and so on. The feature have been tested on latest Chrome, latest Firefox, lastest Safari and IE 11. If possible, please test on Microsoft Edge.

**Comment by MegazeroMZ** [comment]
@brunokrebs since the layout is completely different now, would you mind rewrite the readme.md to reflect the new change? Since after updated to v0.6.0 the ngWYSIWYG completely stop working for me. It dont even show up on the html anymore.

**Comment by brunokrebs** [comment]
Well, I'm going to take a look at that, but the rewrite that I've made should be compatible with previous versions, that's why I published a feature and not a release. Anyway, checking it now.

**Comment by MegazeroMZ** [comment]
Even for the previous version (which I'm reverted back to it for now) the readme should still be updated because of the source file moving. Before it was (and still is on the readme):
bower_components/ngWYSIWYG/css/editor.css
bower_components/ngWYSIWYG/js/wysiwyg.js

But now (for the previous version) it is 
bower_components/ngWYSIWYG/dist/editor.min.css
bower_components/ngWYSIWYG/src/js/wysiwyg.js
bower_components/ngWYSIWYG/src/js/ngpResizable.js

**Comment by brunokrebs** [comment]
I have not tested yet, but considering what you are saying I'm guessing that the problem is that you are adding the wrong files. There are two files that you have to add in your project:

```
dist/editor.min.css
dist/wysiwyg.min.js
```

Both are minified and the second one has all the js files, concatenated in the right order, in the src directory. Please, try with this single js file (alongside with css).

By the way, even that it start working, I still need to update README.md. Thanks for pointing it out.

**Comment by brunokrebs** [comment]
@MegazeroMZ thank you very much for pointing the issues. Now I have updated README.md and, besdides that, I have fixed an issue on editor's style. Take a look at [v0.6.1](https://github.com/psergus/ngWYSIWYG/releases/tag/v0.6.1).

**Comment by MegazeroMZ** [comment]
Thanks. I also found out what was the problem with v0.6.0 for me. I added a custom button in wysiwyg.js file in /src/js folder in the previous version, so I cant read from the wysiwyg.min.js directly since min file dont have the header for my additional function, so when my code from other file try to look for that function in ngWYSIWYG it cant find it, hence the bug. And because of that I couldnt run just the wysiwyg.min.js, but I need to manually included all files in the source. But after that everything work fine.

I found a different bug though, I will post another issue on it.
